lib/helpers.js

```diff
@@ -2,9 +2,12 @@
 
 'use babel';
 
+import Path from 'path';
+import { findCached } from 'atom-linter';
+
 const helpers = require('atom-linter');
 
-export default function check(
+export function check(
```

Comment:
I would not change this from a default export to non-default one.

You can still add additional exports while leaving this be the default. I'm trying to minimise breaking changes here.